src/main.c

```diff
@@ -327,6 +331,13 @@ main(int argc, char* argv[])
     capture_eep_init();
 #endif
 
+    // Split device into tokens and add to list of capture devices
+    device = strtok(device, ",");
+    while (device) {
+        vector_append(indevices, device);
+        device = strtok(NULL, ",");
+    }
+
     // If no device or files has been specified in command line, use default
     if (vector_count(indevices) == 0 && vector_count(infiles) == 0) {
         vector_append(indevices, (char *) device);
```

Comment:
This behavior seems different in the proposed code.

The device string (I think it cames from the config file) is only used if the command line is empty (indevices vector).

With the new code, the device variable is added to the vector no matter if it already contains data. 
I will move this while inside the next if block, replacing the vector_append line.

Also, take into account that you can already use multiple devices from command line using `-d` option multiple times.

Reply:
Oh, I missed that.
I'll fix it.
We can close this pull request, and I'll split the changes in 2 new requests, also